usegalaxy.org/multiple_alignments.yml

```diff
@@ -8,8 +8,6 @@ tools:
   owner: iuc
 - name: mafft
   owner: rnateam
-- name: clustalw
```

Comment:
ClustalW gets a lot of use on Galaxy Australia.

Reply:
same here